lib/msf/core/exploit/remote/kerberos/ticket.rb

```diff
@@ -28,7 +28,7 @@ def create_ticket(enc_key:, start_time:, end_time:, sname:, flags:, domain:, use
               key_value: enc_key,
               checksum_enc_key: enc_key,
               secure_random_key: SecureRandom.hex(8),
-              user_id: 500,
+              user_rid: user_rid,
```

Comment:
This won't work; Where the key is eventually used it requires `user_id` as the option name:

https://github.com/rapid7/metasploit-framework/blob/463915b482991e737e2e5dbdd2b26f4f80312af6/lib/msf/core/exploit/remote/kerberos/client/pac.rb#L47

Reply:
Good catch 👍